### PR TITLE
Disambiguate socket API pathes on Windows with user name

### PR DESCRIPTION
**Pull request by danimo**
Addresses #3411

### CODE REVIEW

File: src/gui/socketapi.cpp [hunk 1]
```diff
@@ -67,7 +67,7 @@ SocketApi::SocketApi(QObject* parent)
 
     if (Utility::isWindows()) {
         socketPath = QLatin1String("\\\\.\\pipe\\")
-        + QLatin1String("ownCloud");
+        + QLatin1String("ownCloud") + '\\' + qgetenv("USERNAME");
```
Comment (ogoffart) [inline src/gui/socketapi.cpp]: I don't know if the username can contains non-ascii, but if that's the case I think this should be  QString::fromLocal8Bit(qgetenv("USERNAME"))

Comment (danimo) [inline src/gui/socketapi.cpp]: Done.

**Comment by guruz** [comment]
👍

**Comment by danimo** [comment]
Wait! This is not tested yet!

**Pull request merged**